config/api.yml

```diff
@@ -1068,6 +1068,56 @@
     - :collection
     :verbs: *g
     :klass: ChargebackRateDetailMeasure
+  :middleware_datasources:
+    :description: Middleware Datasources
+    :klass: MiddlewareDatasource
+    :options:
+      - :collection
+    :verbs: *g
```

Comment:
@cfcosta I think I still need translation :) . what does this section mean in the yml file?